RELEASE.md

```diff
@@ -20,11 +20,9 @@ Releases should be twice monthly<sup>[1](#footnote1)</sup> on the second and for
 
 ### Kubernetes
 
-The Addons in this repo are the minimum base set of supported<sup>[2](#footnote2)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+**The Kubernetes versions supported by this repo are 1.15, 1.16, and 1.17.**
 
-Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
```

Comment:
This is an important restriction that should remain documented.

Reply:
The latest general release of Kubernetes is 1.18, so we're violating this restriction. How about I replace "must" with "should"?

Reply:
@samba This was a product goal from the beginning. Has this changed?

Reply:
I agree that we should always support latest K8s in addition to any other version. However, afaik we're not testing the latest version, are we? So if you asked me we should change the latter and not the former. 😄

Reply:
At least for the moment, changing it to "should" reflects our current situation. 

We should indeed aim to keep up with Kubernetes releases, however they are _not_ guaranteed on a quarterly (3-month) basis, and occasionally shift. 

Recently added 1.17 support in konvoy 1.5, and will be adding 1.18 support soon in konvoy 1.6. I'd like us to catch up with the community when 1.19 is released, but that will depend on several elements of timing. If 1.19 is released before Konvoy 1.6 goes GA, perhaps we can support both in that timeframe -- depending on complexity of migrations.

The present assumption, based on quarterly releases, was that each Konvoy release would target the *next* GA Kubernetes release, i.e. `1.4 -> 1.16`, `1.5 -> 1.17` and so on. As our release schedule flexes, as does Kubernetes upstream, this may not be exactly tenable.  We have deviated from this pattern in previous releases (e.g. Konvoy 1.2, 1.3) but we should be mindful of it in the future, as keeping the pattern would make compatibility guarantees more intuitive.

@chrisgaun frequently reminds me that we have *no customers* asking for Kubernetes 1.18 at this time. Evidently keeping up with the community is not a _customer demand_, but tracking closely with it, especially as charts for addons and other components migrate to newer Kubernetes, will be help us minimize maintenance cost.